sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample6_RecognizeLinkedEntitiesAsync.cs

```diff
@@ -22,12 +21,12 @@ public async Task ExtractEntityLinkingAsync()
 
             string document = "Microsoft was founded by Bill Gates and Paul Allen.";
 
-            Response<IReadOnlyCollection<LinkedEntity>> linkedEntities = await client.RecognizeLinkedEntitiesAsync(document);
+            LinkedEntityCollection linkedEntities = await client.RecognizeLinkedEntitiesAsync(document);
 
-            Console.WriteLine($"Extracted {linkedEntities.Value.Count} linked entit{(linkedEntities.Value.Count > 1 ? "ies" : "y")}:");
-            foreach (LinkedEntity linkedEntity in linkedEntities.Value)
+            Console.WriteLine($"Extracted {linkedEntities.Count} linked entit{(linkedEntities.Count > 1 ? "ies" : "y")}:");
+            foreach (LinkedEntity linkedEntity in linkedEntities)
             {
-                Console.WriteLine($"Name: {linkedEntity.Name}, Language: {linkedEntity.Language}, Data Source: {linkedEntity.DataSource}, Url: {linkedEntity.Url.ToString()}, Entity Id in Data Source: {linkedEntity.DataSourceEntityId}");
+                Console.WriteLine($"Name: {linkedEntity.Name}, Language: {linkedEntity.Language}, Data Source: {linkedEntity.DataSource}, Url: {linkedEntity.Url}, Entity Id in Data Source: {linkedEntity.DataSourceEntityId}");
```

Comment:
Why can you not call .ToString() anymore, I'm not seeing any changes to Url in your PR

Or is this a change that you could've made earlier and decided to make in this PR

Reply:
Compiler was just making suggestions that `.ToString()` was not necessary so I just went and removed it from there.
I honestly didn't look where else the reference was, so I can go ahead and revert this for now if that is more clear.

Reply:
oh no, i don't think that's necessary, just wanted to check